assets/js/util/test/extractForSparkline.js

```diff
@@ -114,8 +114,119 @@ const valuesToTest = [
 	],
 ];
 
+const columnIndexValuesToTest = [
```

Comment:
These are pretty tough to parse and understand, especially because the data is a bit far away from the test. I understand the efficiency in use `it.each` here, but we need to make sure our tests are readable and parsable in the future if we need to change them or diagnose errors. That's why I prefer more verbose tests that don't follow DRY always, because they're easier to reason about. Test code doesn't need to be DRY as much.

That said, I think this is fine to leave as-is, but could you leave comments in the data indicating where the input starts and where the result is? That way someone can look at this `columnIndexValuesToTest` variable and see what the result should be. That'll make this easier to maintain.